test/helpers/browserify.js

```diff
@@ -0,0 +1,16 @@
+var fs = require('fs');
+var sandbox = require('./sandbox');
+
+function exportToFile(b, fileName, callback) {
```

Comment:
Since you are resolving `fileName` to a path in sandbox, how about `exportToSandbox` or `bundleToSandbox`?